src/DGtal/images/ImageContainerByHashTree.h

```diff
@@ -276,6 +276,19 @@ namespace DGtal
      */
     Range range() ;
 
+    /**
+     * Give access to the underlying container.
+     * @return a (might be const) reference to the container.
+    */
+    inline const Self & container() const { return *this; };
```

Comment:
I think inline keyword is redundant with the code placed at the definition (ie inlined).
https://isocpp.org/wiki/faq/inline-functions

Reply:
Good to know, fixed